.ado/jobs/playground.yml

```diff
@@ -119,6 +131,42 @@ jobs:
                   msbuildArgs:
                     /p:PackageCertificateKeyFile=$(Build.SourcesDirectory)\EncodedKey.pfx
 
+            # Execute debug feature tests
+            #
+            # This step loose-file-deploys the UWP Playground app and uses it as an RNW host for a series
+            # of debug feature tests. In the future, these tests should be performed against a host app
+            # better suited for automated tests (probably the E2E test app).
+            - powershell: |
+                $appRecipeToolPath = Join-Path (Get-CimInstance MSFT_VSInstance | Sort-Object -Property 'Version' -Descending)[0].InstallLocation "Common7\IDE\DeployAppRecipe.exe"
```

Comment:
note you can just use `vswhere` to locate VS installs, no need for WMI :)

Reply:
I'm aware of vswhere (vswhere-aware?). Is it in the path of powershell instances on the ADO agents? I would expect this to be the case only inside VS dev prompts - which I would imagine are brought about via a .bat file that doesn't really apply to Powershell?

Reply:
it is guaranteed to be in %ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe